superdesk/sams/assets.py

```diff
@@ -148,3 +148,25 @@ def get_assets_compressed_binary(asset_ids):
         item_ids=asset_ids
     )
     return zip_binary.content
+
+
+@assets_bp.route("/sams/assets/lock/<asset_id>", methods=["PATCH"])
+def lock_asset(asset_id):
+    docs = {"lock_action": "edit"}
```

Comment:
`lock_action` should be provided by the front-end (later we may add further actions on Assets that will require a different lock action)

Reply:
Okay.